ingestion-beam/bin/download-sampled-landfill.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
```

Comment:
Please add a module docstring explaining the purpose and usage of this executable.